pages/features.vue

```diff
@@ -318,6 +318,12 @@
           Families in premium campaign have access to beautiful family trees.
         </p>
       </div>
+      <div id="webhooks" class="flex flex-col gap-5">
+        <h3>Webhooks</h3>
+        <p>
+          Users of premium campaigns can create custom webhook updates that notify them on a third party application like Discord whenever changes to the campaign are made.
```

Comment:
"Premium campaigns can define workflows that send updates to Discord and other tools through webhooks when changes to entities are made."
Also missing in the "table" view of premium features.